src/Lean/Server/AsyncList.lean

```diff
@@ -87,33 +87,23 @@ partial def waitFind? (p : α → Bool) : AsyncList ε α → BaseIO (Task $ Exc
   | cons hd tl => do
     if p hd then return Task.pure <| Except.ok <| some hd
     else tl.waitFind? p
-  | asyncTail tl => do
+  | delayed tl => do
     BaseIO.bindTask tl fun
       | Except.ok tl   => tl.waitFind? p
       | Except.error e => return Task.pure <| Except.error e
 
-/-- Extends the `finishedPrefix` as far as possible. If computation was ongoing
-and has finished, also returns the terminating value. -/
-partial def updateFinishedPrefix : AsyncList ε α → BaseIO (AsyncList ε α × Option ε)
+/-- Compute the already-computed prefix of the list. If computation has finished with an error, return it as well. -/
```

Comment:
```suggestion
/-- Return the already-computed prefix of the list. If computation has finished with a terminating value,
return it as well. -/
```
Since `ε` is the type of terminating values that can be either an error or a successful end-of-computation signal.

Reply:
Right, I was wondering about that part in the previous docstring. Do you mean `ElabTaskError.aborted` by this? I don't really think of it as successful termination.